src/callbacks/onDraw/updateStudyDayControl.js

```diff
@@ -0,0 +1,17 @@
+export default function updateStudyDayControl() {
+    var config = this.config;
+    const studyDayControlWrap = this.controls.wrap
+        .selectAll('div')
+        .filter(controlInput => controlInput.label === 'Study Day');
+
+    // hide study day control if viewing max values
+    studyDayControlWrap.style('display', config.plot_max_values ? 'none' : null);
+
+    //update the study day control label with the currently selected values
+    const currentValue = studyDayControlWrap.select('input').property('value');
```

Comment:
This value gets rounded to the nearest multiple of 7, counting from the minimum study day in the data, because of the `step` attribute.  Not immediately obvious why that is.

Reply:
It was my hack-y way of using week-long increments ...